lib/ops/opsservice/shrink.go

```diff
@@ -146,7 +146,7 @@ func (s *site) validateShrinkRequest(req ops.CreateSiteShrinkOperationRequest, c
 	teleserver := servers.getWithLabels(labels{ops.Hostname: server.Hostname})
 	if len(teleserver) == 0 {
 		if !req.Force {
-			return nil, trace.Wrap(err,
+			return nil, trace.BadParameter(
```

Comment:
This is the bug we fixed in 5.0.x but not in master. Basically, when removing an offline server w/o `--force`, the code would panic instead of displaying this error message.